contrib/babelfishpg_tsql/src/session.c

```diff
@@ -70,6 +70,7 @@ void
 set_cur_db_name_for_parallel_worker(const char* logical_db_name)
```

Comment:
nit: Let's update the function name to reflect we are setting db_id as well.

Reply:
I don't think that is really needed here.